src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: blanchedalmond;
+}
+
+.container {
+  background-color: bisque;
+  width: 40%;
+  margin: auto;
+  margin-top: 70px;
+}
+
+.header {
+  text-align: center;
+}
+
+.list li {
+  margin-top: 2px;
+  list-style-type: square;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement the correct design [provided by Microverse](https://www.youtube.com/watch?v=AcUd-_Yjjqg) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/205978283-b326bbc2-f571-4f5b-b904-7de24f0a1d5d.png)